cmd/devtool/devtool.go

```diff
@@ -69,11 +73,25 @@ func main() {
 	}
 	if !goodToGo {
 		fmt.Println(`
-    Usage: go run cmd/devtool/devtool.go setup broadcaster|transcoder
+    Usage: go run cmd/devtool/devtool.go setup broadcaster|transcoder [nodeIndex]
         It will create initilize eth account (on private testnet) to be used for broadcaster or transcoder
-        and will create shell script (run_broadcaster_ETHACC.sh or run_transcoder_ETHACC.sh) to run it.`)
+				and will create shell script (run_broadcaster_ETHACC.sh or run_transcoder_ETHACC.sh) to run it.
+				Node index indicates how much to offset node's port. Orchestrator node's index by default is 1.`)
```

Comment:
Not quite clear on what an "node index" is supposed to do, maybe an example would help or some deeper documentation of each of devtool's options elsewhere (eg, in `/docs/devtool.md`).

Reply:
Added example